### PR TITLE
Add landing message. #1

### PR DESCRIPTION
**Pull request by dragan-pecanac**
added welcome message

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,6 @@
     <title>Welcome to Potionfy</title>
   </head>
   <body>
-    
+    <h1>Welcome to Potionfy! Enjoy your stay.</h1>
```
Comment (Crouching-Dragan) [inline index.html:10]: Could you remove the 'Enjoy your stay.' string and make the 'Welcome to Potionfy!' string red?

Comment (dragan-pecanac) [reply index.html:10]: Sure, can do!

**Review by Crouching-Dragan — CHANGES_REQUESTED** [review]
Please, change the code as per my comments on the review.

**Review by Crouching-Dragan — APPROVED** [review]
Well done!

**Pull request merged**